R/status.R

```diff
@@ -124,24 +124,32 @@ renv_status_impl <- function(project, libpaths, lockpath, sources, cache) {
   has_lockfile <- file.exists(lockpath)
 
   if (!has_library || !has_lockfile) {
+
     if (!has_library && !has_lockfile) {
       writef(c(
-        "Project does not use renv.",
-        "Call renv::init() to setup project to use renv."
+        "This project does not appear to be using renv.",
+        "Use renv::init() to initialize this project."
       ))
     } else if (!has_library) {
       writef(c(
-        "Project lacks a library.",
-        "Call renv::restore() to install packages defined in lockfile."
+        "This project does not have a private library.",
```

Comment:
Is private the right word?

Reply:
Yeah, maybe not -- that was the older nomenclature. I guess "The project library does not exist." or something else like that?